pkg/cmd/dev/build.go

```diff
@@ -71,6 +71,7 @@ func makeBuildCmd(runE func(cmd *cobra.Command, args []string) error) *cobra.Com
 
 // buildTargetMapping maintains shorthands that map 1:1 with bazel targets.
 var buildTargetMapping = map[string]string{
+	"all_tests":        "//pkg:all_tests",
```

Comment:
`dev build tests` (the alias we added below) feels more than sufficient. I imagine we'd only be using this to ensure that code changes compile across test files -- I don't see much value in exposing `{large,medium,small}_test` here (especially given it's it's just a `dev build //pkg:large_test` away); if anything it clutters things.